packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -65,10 +95,16 @@ contract SystemConfig is OwnableUpgradeable, Semver {
     bytes32 public batcherHash;
 
     /**
-     * @notice L2 gas limit.
+     * @notice L2 block gas limit.
      */
     uint64 public gasLimit;
 
+    /**
+     * @notice The configuration for the deposit fee market. Used by the OptimismPortal
+     *         to meter the cost of buying L2 gas on L1.
```

Comment:
Worth adding a comment here as to why this is internal. I know it's explained in the external function but nice to have here too.